style.css

```diff
@@ -0,0 +1,582 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+body {
+  background-color: rgb(247, 240, 240);
+}
+
+.covers {
+  width: auto;
+  margin-bottom: 100px;
+  margin-top: 100px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background-color: white;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background-size: cover;
+  border-radius: 20px;
+}
+
+.img__class {
+  width: auto;
+  height: 220px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 16px;
+  margin-bottom: 24px;
+  border-radius: 20px;
+}
+
+.img__class:hover {
+  border: 2px solid #396df2;
+}
+
+.bold-text {
+  width: auto;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 20px;
+  font-family: poppins, sans-serif;
```

Comment:
- [OPTIONAL] To keep your css file dry, please use classes for properties that are repeating.